cpu/sam0_common/include/periph_cpu_common.h

```diff
@@ -69,6 +87,20 @@ typedef uint32_t gpio_t;
 
 #ifndef DOXYGEN
 /**
+ * @brief   Override GPIO modes
+ * @{
```

Comment:
The opening block `@{` is not needed with `@brief`

Reply:
Why is the doxygen docs inside ifndef DOXYGEN?

Reply:
I've no idea... I just copy&paste from SAMD21 periph_cpu file
Should I remove them @gebart ?

Reply:
Doesn't matter, I just thought it was a funny curiosity. I know that the ifndef is there to prevent multiple copies of the same enum symbol with different values in the generated docs.